ENIGMAsystem/SHELL/Universal_System/roomsystem.cpp

```diff
@@ -234,8 +234,12 @@ namespace enigma
     //if the window size changes.
     enigma_user::window_default(true);
     enigma_user::io_clear();
-    screen_init();
-    screen_refresh();
+    // we only initialize the screen and clear the window color during game start
+    // NOTE: no version of GM has EVER reset the drawing color or alpha during room transition
+    if (gamestart) {
```

Comment:
I used to find out that you @JoshDreamland and @rpjohnst are responsible for adding `gamestart` as a parameter.
You two made this change in 5e488a111fb71c5a3d0baed02e2793e81f671299

It seems that, with this pr, we are using `gamestart` for three things:
1)
2)
3)

Reply:
@JoshDreamland let's not conflate this here, but you may have a point. I don't think it's actually a matter of the logic being coupled, I think it's just more of a time-constraint thing why some of this game start logic is in here to begin with, possibly, I'm not sure. Regardless, sometime in the nearer future, it may be a good idea if we clean this up sooner rather than later before it becomes a big mess like is the typical thing that happens when we let something go for too long.